EkaterinaDubinina.iml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4" />
```

Comment:
Why IDE related file in the commit?

Reply:
All IDE related files added to the gitignore.